button/popup.css

```diff
@@ -292,11 +296,11 @@ http://matthewlein.com/ceaser/
 }
 
 .sitetitle {
-    margin-top: 3px;
     display: block;
-    left: -10px;
+    height: 35px;
     position: inherit;
-    padding: 10px 0 0 15px;
+    margin: 15px 10px 0px 5px;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/tomastaro/notemark/file/2651364002/issues/source?bid=2884821&fileBranchId=3324709#l302)